src/Button/ToggleGroup/ToggleGroup.jsx

```diff
@@ -169,12 +156,8 @@ class ToggleGroup extends React.Component {
     const orientationClass = (orientation === 'vertical')
       ? 'vertical-toggle-group'
       : 'horizontal-toggle-group';
-
     const childrenWithProps = React.Children.map(children, (child) => {
-      // Only set pressed state for ToggleButtons
-      if (!(child.type === ToggleButton)) {
-        return child;
-      }
```

Comment:
Why did you remove this check?

Reply:
This wasn't passing the pressed property to other components that were not `Togglebutton`s like the `Digitizebutton` which is a component made of `togglebuttons` and the `pressed` property needs to gets pass through to it.